src/Form/RadioGroup.js

```diff
@@ -60,7 +67,7 @@ class RadioGroup extends Component {
   };
 
   render() {
-    const { choices, error, horizontal, label, name, colorOn, colorOff, size } = this.props;
+    const { choices, error, horizontal, label, name, colorOn, colorOff, ...checkboxProps } = this.props;
```

Comment:
colorOn and colorOff and be swept into checkboxProps spread

Reply:
OK, removed.